src/Config/Requirement/RequirementViolationList.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Config\Requirement;
+
+class RequirementViolationList
```

Comment:
👍  but let's let @xavier-rdo do its magic and come back when `[WIP]` is no more in the title :)